lib/xcodebuild.js

```diff
@@ -238,9 +238,18 @@ class XcodeBuild {
         'Will build for the default platform instead');
     }
 
-    if (this.realDevice && this.xcodeConfigFile) {
-      this.log.debug(`Using Xcode configuration file: '${this.xcodeConfigFile}'`);
-      args.push('-xcconfig', this.xcodeConfigFile);
+    if (this.realDevice) {
+      if (this.xcodeConfigFile) {
+        this.log.debug(`Using Xcode configuration file: '${this.xcodeConfigFile}'`);
+        args.push('-xcconfig', this.xcodeConfigFile);
+      }
+      if (this.xcodeOrgId && this.xcodeSigningId) {
+        args.push(
+          `DEVELOPMENT_TEAM=${this.xcodeOrgId}`,
+          `CODE_SIGN_IDENTITY=${this.xcodeSigningId}`,
+          'CODE_SIGN_STYLE=Manual'
```

Comment:
```suggestion
```

it seems like this should not exist. Build failed as 70 with `CODE_SIGN_STYLE=Manual`, but succeeded without that. Maybe CODE_SIGN_STYLE=Manual requires more something.


Btw, when the build failure occurred, the below check continued even response back the error message to the client. Maybe we need to tweak that later

```
[debug] [XCUITestDriver@6bc0] Proxying [GET /status] to [GET http://127.0.0.1:8100/status] with no body
[XCUITestDriver@6bc0] connect ECONNREFUSED 127.0.0.1:8100
[debug] [XCUITestDriver@6bc0] Matched '/status' to command name 'getStatus'
[debug] [XCUITestDriver@6bc0] Proxying [GET /status] to [GET http://127.0.0.1:8100/status] with no body
[XCUITestDriver@6bc0] connect ECONNREFUSED 127.0.0.1:8100
[debug] [XCUITestDriver@6bc0] Matched '/status' to command name 'getStatus'
[debug] [XCUITestDriver@6bc0] Proxying [GET /status] to [GET http://127.0.0.1:8100/status] with no body
[XCUITestDriver@6bc0] connect ECONNREFUSED 127.0.0.1:8100
[debug] [XCUITestDriver@6bc0] Matched '/status' to command name 'getStatus'
[debug] [XCUITestDriver@6bc0] Proxying [GET /status] to [GET http://127.0.0.1:8100/status] with no body
```

Reply:
^ tested with xcode 15 and 14.3.1

Reply:
sure, thanks for checking